### PR TITLE
Methods for setting/reseting votecode. Removing unused code

### PR DESCRIPTION
**Pull request by navro**

### CODE REVIEW

File: app/views/layouts/_admin_dropdown.html.erb [hunk 1]
```diff
@@ -19,43 +19,33 @@
...
       <li class="dropdown-submenu">
-        <%= link_to 'Voteringar', admin_votes_path %>
+        <%= link_to models_name(Vote), admin_votes_path %>
       </li>
     <% end %>
-    <% if can? :manage, User %>
+    <% if can? :manage_voting, User %>
       <li class="dropdown-submenu">
-        <%= link_to models_name(User), admin_users_path %>
+        <%= link_to models_name(User), admin_vote_users_path %>
```
Comment (davidwessman) [inline app/views/layouts/_admin_dropdown.html.erb]: This should perhaps be named something other than normal `Users`, like `user.vote_administration`.

File: app/views/static_pages/about.html.erb [hunk 2]
```diff
@@ -1,12 +1,7 @@
...
+<h2><%= t(:about) %></h2>
 <div class="search-blocks search-blocks-top-orange">
   <div class="col-md-8 col-xs-12">
-    <%= simple_format(t(:about_us_first))%>
-    <%= simple_format(t(:about_us_second))%>
-    <%= simple_format(t(:about_us_third))%>
-  </div>
-  <div class="col-md-4 col-xs-12">
-    <%= render Menu.where(location: 'guild') %>
+    Nu har vi äntligen ett digitalt valsystem!
```
Comment (davidwessman) [inline app/views/static_pages/about.html.erb]: I18n.translate is worth it! 😄

File: config/routes.rb [hunk 3]
```diff
@@ -44,6 +44,7 @@
       resources :vote_users, path: :motesanvandare, controller: :vote_users do
         get 'change_state', on: :member
         get 'make_all_not_present', on: :collection
+        get 'new_votecode', on: :member
```
Comment (davidwessman) [inline config/routes.rb]: I feel like these actions should be `patch`.

- `GET` is mostly used for loading a page
- `POST` for creating new content
- `PATCH` for updating existing content

**Comment by davidwessman** [comment]
Just a few comments, great job! 😄

File: app/views/static_pages/about.html.erb (hunk 2 above)
Comment (navro) [inline app/views/static_pages/about.html.erb]: I agree, but this is just a placeholder :) We are probably going to change this page anyway

Comment (davidwessman) [inline app/views/static_pages/about.html.erb]: Sounds valid 😄

**Pull request merged**